UnityProject/Assets/Scripts/Objects/StatusDisplay.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections;
+using Mirror;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+/// <summary>
+/// Status display that will show short text messages sent to currently selected channel.
+/// Escape Shuttle channel is a priority one and will overtake other channels.
+/// </summary>
+public class StatusDisplay : NetworkBehaviour, IOffStageServer, IOnStageServer
+{
+	public static readonly int MAX_CHARS_PER_PAGE = 18;
+
+	private Coroutine blinkHandle;
+
+	[SerializeField]
+	private Text textField;
+
+	[SyncVar(hook = nameof(SyncStatusText))]
+	private string statusText;
+
+	public StatusDisplayChannel Channel
+	{
+		get => channel;
+		set
+		{
+			cachedText = string.Empty;
+			channel = value;
+		}
+	}
+	/// <summary>
+	/// don't change it in runtime, use Channel property instead
+	/// </summary>
+	[SerializeField]
+	private StatusDisplayChannel channel = StatusDisplayChannel.Command;
+
+	/// <summary>
+	/// used for restoring selected channel message when priority message is gone
+	/// </summary>
+	private string cachedText;
+
+	#region syncvar boilerplate and init
+
+	public override void OnStartClient()
+	{
+		SyncStatusText();
```

Comment:
change this to SyncStatusText(statusText)

Reply:
I tried it before and got NREs on clientside, I think statusText is null at this point

Reply:
Weird, I just changed the code to make sure it works and it was fine. This was client side too